include/swift/Runtime/OperatorNew.h

```diff
@@ -0,0 +1,65 @@
+//===-- OperatorNew.h - Definitions of operators new and delete -*- C++ -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+// Never use the global definitions of operators new and delete.
+// The app may replace them and we don't trust their implementations.
+
+#include <new>
+#include <stdlib.h>
+
+#define SWIFT_PRIVATE_INLINE \
+  __private_extern__ inline __attribute__((always_inline))
+
```

Comment:
I don't see any existing wrappers around __private_extern__ in llvm or swift. It's possible that __private_extern__ is unnecessary given always_inline.